app.js

```diff
@@ -0,0 +1,93 @@
+const form = document.querySelector('#new-task-form');
+const button = document.querySelector('#button');
+const list =  document.querySelector('#task-list');
+const field = form['new-task-input'];
+
+
+const todoList = [];
+
+createEditButton = () => {
+    const editButton = document.createElement('button');
+    editButton.classList.add = 'edit';
+    editButton.innerText = 'Edit';
+
+    //edit task handler
+    editButton.addEventListener('click', (e) => {
+        //logic
+    })
+
+    return editButton;
+}
+
+createDeleteButton = () => {
+    const deleteButton = document.createElement('button');
+    deleteButton.classList.add = 'delete';
+    deleteButton.innerText = 'Delete';
+
+    //delete task handler
```

Comment:
Vague comment. Only add comment where the context is absolutely necessary. Other than that, the code itself should say what it does